mesheryctl/internal/cli/root/perf/result.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
 	"github.com/layer5io/meshery/models"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
we want to move away from logus completely. pls use utils.Log instead. Ref. #5298

Reply:
Done 👍